docs/spark-connect-overview.md

```diff
@@ -113,14 +113,15 @@ Now Spark server is running and ready to accept Spark Connect sessions from clie
 applications. In the next section we will walk through how to use Spark Connect
 when writing client applications.
 
-## Use Spark Connect in client applications
+## Use Spark Connect for Interactive Analysis
```

Comment:
Since this section covers both using Spark Connect in a shell (interactive analysis) and when writing a program, can we just name this section "How to use Spark Connect" or similar?

Reply:
Sure

Reply:
> shell (interactive analysis) and when writing a program

Note that this only applies to Python. For scala, a program not launched as a REPL has different setup requirements